cypress.config.ts

```diff
@@ -8,7 +8,7 @@ export default defineConfig({
   projectId: 'eup7bh',
   e2e: {
     specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
-    baseUrl: 'http://localhost:8081/',
```

Comment:
Why does this need to be changed?

Reply:
For the CI. Something about using localhost was causing the ui to be unreachable by cypress